tests/conf/api_acc_model_conf.py

```diff
@@ -0,0 +1,51 @@
+import time
+import os
+import random
+import datetime
+
+current_timestamp = str(int(time.time()))
+
+# Bearer token from environment variables
+bearer_token = os.environ.get('bearer_token')
+
+# ACC model details and attribute details
+acc_details = {'name': 'Newsletter App'+current_timestamp, 'description': 'Creating an acc model for newsletter app'}

```

Comment:
Improvise the description. Make it related to what the app is about like "An app for scheduling weekly newsletter for Qxf2"